apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -1081,6 +1081,37 @@ function handleInlineHelpButton( calypsoPort ) {
 	);
 }
 
+/**
+ * If WelcomeTour is set to show, check if the App Banner is visible.
+ * If App Banner is visible, we set the Welcome Tour to not show.
+ * When the App Banner gets dismissed, we set the Welcome Tour to show.
+ *
+ * @param {MessagePort} calypsoPort Port used for communication with parent frame.
+ */
+function getIsAppBannerVisible( calypsoPort ) {
```

Comment:
I wonder if a different name here would work a little better. It might just be me, but it feels like the description (and hence logic) and the name `getIsAppBannerVisible` don't really serve each other well. We are effectively "handling" something here right? I wonder if this is like a naming convention that propagated (maybe for wrong reasons, cause other conventions appear in this file too?).

Reply:
I think you're right. It propagated because, _in_ the beginning, I was using the `calypsoifyGutenberg` global variable, and so I followed what `getGutenboardingStatus` was doing. In the current approach we are "handling", as you said, the App Banner dismissing, so I will name it better 👍